cmd/time-box/start.go

```
@@ -0,0 +1,71 @@
+			{Label: "Work", Duration: time.Duration(workMin) * time.Minute},
+			{Label: "Break", Duration: time.Duration(breakMin) * time.Minute},
+		}
+
+		ch := make(chan timer.Tick, 1)
+		go timer.Run(ctx, phs, cycles, ch)
+
+		for t := range ch {
+			fmt.Printf("\r%-5s %02d:%02d", t.Phase,
+				int(t.Left.Minutes()), int(t.Left.Seconds())%60)
+			if t.Left == 0 {
+				notify.Push("time-box", t.Phase+" done!")
```

golangci-lint (1.64.8)

Error return value of `notify.Push` is not checked

(errcheck)